CMakeLists.txt

```diff
@@ -206,6 +266,13 @@ target_include_directories(${PROJECT_NAME}
         $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src/c>
     )
 
+if(PLATFORM_EXTERNAL)    
+    target_include_directories(${PROJECT_NAME} PUBLIC
+        ${PLATFORM_EXTERNAL_TRANSPORT_HEADER_DIRECTORY}
+        $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src/c/profile/transport/serial>
```

Comment:
Why do you add an src file as an include directory?

Reply:
I need to have access to serial_transport_internal.h in order to provide an external transport. This file is in folder /src/c/profile/transport/serial.